set.mm

```diff
@@ -10475,6 +10475,14 @@ This definition (in the form of ~ dfifp2 ) appears in Section II.24 of
       ( 3exp2 imp41 ) ABCDEABCDEFGH $.
   $}
 
+  $( An equivalence of two four-terms conjunctions with the terms regrouped
+     (here, the second sub-conjunct of the first term is pulled separately).
+     (Contributed by Zhi Wang, 4-Sep-2024.) $)
+  4aneqpull12 $p |- ( ( ( ph /\ ps ) /\ ( ch /\ th ) )
```

Comment:
`4...2` or `22...12`, so I think maybe

`4anpull2` (note: eq would be suffix "b" but it's probably not necessary)